<a href="https://colab.research.google.com/github/dtht2d/scintilator/blob/main/codes/PbWO4_monoclinic.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# **Electronic Structure of  $\text{PbWO}_4$ and optical properties of Scintillator Materials** 
---

In [46]:
%autosave 30

Autosaving every 30 seconds


## Abstract

## Objectives



Study the electronic structure and optical properties of $\text{PbWO}_4$

What makes Lead Tungstate (PbWO4) crystal a popular choice as a detector material in high energy physics devices, such as the CMS detector at CERN?

Is its quick decay time, high density, and strong resistance to radiation. Its unique properties also make it a promising material for use in medical devices. In addition, it is widely used in photonics applications due to its stable physical and chemical properties and exceptional optical properties.

Study the principle of nano-photonics and how we can integrate nano-photonics with traditional scintillator materials to improve the development of radiation detection

### Install packages

In [47]:
!pip install mp-api

Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/


In [71]:
!pip install vtk

Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/


In [59]:
pip install pymatgen[vis]

Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/


In [60]:
!pip install --upgrade pymatgen

Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/


In [89]:
!pip install ase

Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.2/2.2 MB 55.1 MB/s eta 0:00:00


for visualization

In [97]:
!wget http://jp-minerals.org/vesta/archives/3.4.4/vesta-3.4.4-linux-x86_64.tar.bz2
!tar -xvf vesta-3.4.4-linux-x86_64.tar.bz2

--2023-03-24 00:12:20--  http://jp-minerals.org/vesta/archives/3.4.4/vesta-3.4.4-linux-x86_64.tar.bz2
Resolving jp-minerals.org (jp-minerals.org)... 157.7.107.91
Connecting to jp-minerals.org (jp-minerals.org)|157.7.107.91|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2023-03-24 00:12:21 ERROR 404: Not Found.

tar: vesta-3.4.4-linux-x86_64.tar.bz2: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now


## **I. Background**
---


## **II. Computational Methods**
---


## **1. Data**

### a. Get structure of Lead tungstate $\text{PbWO}_4$ 
Using The material project database


In [50]:
from mp_api.client import MPRester
# Create an instance of the MPRester class with API key
api_key = "2o5h5xcZkd72tk78L2JoYIhfAt7SiBA3"
with MPRester(api_key) as mpr:
    # You can pass in a formula to get_materials_ids
    list_PbWO4= mpr.get_materials_ids("PbWO4") 
print (list_PbWO4)

Retrieving MaterialsDoc documents:   0%|          | 0/10 [00:00<?, ?it/s]

[MPID(mp-22410), MPID(mp-504591), MPID(mp-558734), MPID(mp-641365), MPID(mp-641415), MPID(mp-641441), MPID(mp-689140), MPID(mp-795689), MPID(mp-1173387), MPID(mp-1179954)]


Choose: MPID(mp-641365)

In [83]:
with MPRester(api_key) as mpr:
    docs = mpr.summary.search(material_ids=["mp-641415"], fields=["structure"])
    # Retrieve the crystal structure data for the material with ID "mp-641415"
    structure_data = mpr.get_structure_by_material_id("mp-641415")
print(structure_data)

/usr/local/lib/python3.9/dist-packages/mp_api/client/mprester.py:146: UserWarning: mpcontribs-client not installed. Install the package to query MPContribs data, or construct pourbaix diagrams: 'pip install mpcontribs-client'
  warnings.warn(


Retrieving SummaryDoc documents:   0%|          | 0/1 [00:00<?, ?it/s]

Retrieving MaterialsDoc documents:   0%|          | 0/1 [00:00<?, ?it/s]

Full Formula (Pb8 W8 O32)
Reduced Formula: PbWO4
abc   :   7.180633   7.495824  14.839475
angles:  60.011400  90.000000  90.000000
pbc   :       True       True       True
Sites (48)
  #  SP           a         b         c    magmom
---  ----  --------  --------  --------  --------
  0  Pb    0.796834  0.984856  0.346013        -0
  1  Pb    0.703166  0.984856  0.846013        -0
  2  Pb    0.203166  0.015144  0.653987        -0
  3  Pb    0.296834  0.015144  0.153987        -0
  4  Pb    0.539277  0.519534  0.354399        -0
  5  Pb    0.960723  0.519534  0.854399        -0
  6  Pb    0.460723  0.480466  0.645601        -0
  7  Pb    0.039277  0.480466  0.145601        -0
  8  W     0.332694  0.997578  0.41875          0
  9  W     0.167306  0.997578  0.91875          0
 10  W     0.667306  0.002422  0.58125          0
 11  W     0.832694  0.002422  0.08125          0
 12  W     0.042195  0.433783  0.41325          0
 13  W     0.457805  0.433783  0.91325          0
 14  W     0.9578

### b. Write structure data to CIF and visualize $\text{PbWO}_4$ crystal system 

In [84]:
from google.colab import drive
drive.mount('/content/drive')

Mounted at /content/drive


In [95]:
path = "/content/drive/MyDrive/physics5535"
cif_path =path + "/PbWO4.cif" 
# Write the CIF file to disk
structure_data.to(filename=cif_path)

In [96]:
from ase.io import read
from ase.visualize import view
structure = read('/content/drive/MyDrive/physics5535/PbWO4.cif')
view(structure)

<Popen: returncode: None args: ['/usr/bin/python3', '-m', 'ase', 'gui', '-']>

### c. Material description

$\text{PbWO}_4$ crystalinzes in the monoclinic $\text{P2}_1/\text{c}$ space group

| Parameter | Value |
| --- | --- |
| Number of atoms | 48 |
| Crystal system | Monoclinic |
| Lattice system | Monoclinic |
| Symbol |  $\text{P2}_1/\text{c}$  |
| Point group | 2/m
| Possible Ocxidation States | $\text{W}^ {6+}, \ \text{Pb}^ {2+}, \ \text{O}^ {2-}$ 
| Cell length a | 7.180633 Å |
| Cell length b | 7.495824 Å |
| Cell length c | 14.839475 Å |
| Angle α | 60.011400° |
| Angle β | 90.000000° |
| Angle γ | 90.000000° |
| Volume | 691.8009357914264 Å |

### Plotting

## References

---

[1][https://iopscience.iop.org/article/10.1088/1748-0221/5/03/P03010/pdf](https://iopscience.iop.org/article/10.1088/1748-0221/5/03/P03010/pdf)

[2][https://umkc-illiad-oclc-org.proxy.library.umkc.edu/illiad/illiad.dll?Action=10&Form=72&Value=524188](https://umkc-illiad-oclc-org.proxy.library.umkc.edu/illiad/illiad.dll?Action=10&Form=72&Value=524188)

[3][http://cds.cern.ch/record/687201/files/note00_002.pdf](http://cds.cern.ch/record/687201/files/note00_002.pdf)

[4][https://www.photonics.com/Articles/Scintillator_Detectors_with_Integrated/a57310](https://www.photonics.com/Articles/Scintillator_Detectors_with_Integrated/a57310)

[5][http://jdj.mit.edu/~chrc/scintillation.html](http://jdj.mit.edu/~chrc/scintillation.html)

[6][https://arxiv.org/pdf/2103.13106.pdf](https://arxiv.org/pdf/2103.13106.pdf)

[7][https://www.researchgate.net/publication/241252047_Comparative_study_of_electronic_structures_and_nonproportionality_of_scintillator_materials](https://www.researchgate.net/publication/241252047_Comparative_study_of_electronic_structures_and_nonproportionality_of_scintillator_materials)

[8][https://arxiv.org/ftp/arxiv/papers/1509/1509.07822.pdf](https://arxiv.org/ftp/arxiv/papers/1509/1509.07822.pdf)

Computation
---
[1] https://workshop.materialsproject.org/lessons/04_materials_api/MAPI%20Lesson%20%28filled%29/